src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/update/OverseasEntityUpdateSubmissionDtoValidatorTest.java

```diff
@@ -0,0 +1,201 @@
+import static org.mockito.Mockito.verify;
+import static uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto.*;
+
+@ExtendWith(MockitoExtension.class)
+class OverseasEntityUpdateSubmissionDtoValidatorTest {
+    private static final String LOGGING_CONTEXT = "123456";
+
+    private static final String LONG_NAME = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABB";
+
+    @InjectMocks
+    private OverseasEntitySubmissionDtoValidator overseasEntitySubmissionDtoValidator;
+    @Mock
+    private EntityNameValidator entityNameValidator;
+    @Mock
+    private EntityDtoValidator entityDtoValidator;
+    @Mock
+    private PresenterDtoValidator presenterDtoValidator;
+    @Mock
+    private OverseasEntitySubmissionDto overseasEntitySubmissionDto;
+    @Mock
+    private DueDiligenceDataBlockValidator dueDiligenceDataBlockValidator;
+    @Mock
+    private OwnersAndOfficersDataBlockValidator ownersAndOfficersDataBlockValidator;
+    private final EntityNameDto entityNameDto = EntityNameMock.getEntityNameDto();
+    private final EntityDto entityDto = EntityMock.getEntityDto();
+    private final PresenterDto presenterDto = PresenterMock.getPresenterDto();
+    private final OverseasEntityDueDiligenceDto overseasEntityDueDiligenceDto = OverseasEntityDueDiligenceMock.getOverseasEntityDueDiligenceDto();
+    private final DueDiligenceDto dueDiligenceDto = DueDiligenceMock.getDueDiligenceDto();
+    private final BeneficialOwnersStatementType beneficialOwnersStatement = BeneficialOwnersStatementType.NONE_IDENTIFIED;
+    private final List<BeneficialOwnerIndividualDto> beneficialOwnerIndividualDtoList = new ArrayList<>();
+    {
+        beneficialOwnerIndividualDtoList.add(BeneficialOwnerAllFieldsMock.getBeneficialOwnerIndividualDto());
+    };
+    private final List<BeneficialOwnerCorporateDto> beneficialOwnerCorporateDtoList = new ArrayList<>();
+    {
+        beneficialOwnerCorporateDtoList.add(BeneficialOwnerAllFieldsMock.getBeneficialOwnerCorporateDto());
+    };
+    private final List<BeneficialOwnerGovernmentOrPublicAuthorityDto> beneficialOwnerGovernmentOrPublicAuthorityDtoList = new ArrayList<>();
+    {
+        beneficialOwnerGovernmentOrPublicAuthorityDtoList.add(BeneficialOwnerAllFieldsMock.getBeneficialOwnerGovernmentOrPublicAuthorityDto());
+    };
+
+    private final List<ManagingOfficerIndividualDto> managingOfficerIndividualDtoList = new ArrayList<>();
+    {
+        managingOfficerIndividualDtoList.add(ManagingOfficerMock.getManagingOfficerIndividualDto());
+    };
+    private final List<ManagingOfficerCorporateDto> managingOfficerCorporateDtoList = new ArrayList<>();
+    {
+        managingOfficerCorporateDtoList.add(ManagingOfficerMock.getManagingOfficerCorporateDto());
+    };
+
+    @Test
+    void testRegistrationSubmissionCalledWhenEntityNumberIsNullAndUpdateFlagTrue() {
```

Comment:
I'm a bit confused. If entity number is null then it's a registration not update so wondering why this code is here in the `OverseasEntityUpdateSubmissionDtoValidatorTest` class. It's probably duplicating a test that is already in the `OverseasEntitySubmissionDtoValidatorTest` class, no?